build-system/pr-check/checks.js

```diff
@@ -81,6 +81,10 @@ async function prBuildWorkflow() {
     timedExecOrDie('amp ava');
   }
 
+  if (buildTargetsInclude(Targets.BUILD_SYSTEM)) {
+    timedExecOrDie('amp check-build-system');
```

Comment:
This command will also need to be added to `pushBuildWorkflow()` so it runs during `main` branch builds. Perhaps between lines 35 and 36 to maintain ordering?